SUMMARY.md

```diff
@@ -0,0 +1 @@
+docs/README.md
```

Comment:
Didn't appear to work, good pickup :) I moved the docs/README.md into the root and renamed to SUMMARY.md (the GitBook default name for Table of Contents).